packages/acgtk/acgtk.1.5.1/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+maintainer: "sylvain.pogodalla@inria.fr"
+
+build: [
+  ["dune" "subst"] {pinned}
+# remove the -p to also build the local libraries: conflict with the
+# fact that some libraries are also part of the acgtkLib package
+#  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "build" "-j" jobs]
```

Comment:
```suggestion
  ["dune" "build" "--profile=release" "-j" jobs]
```
If you don't use `-p` at least you need `--profile=release`, otherwise dune will treat some warnings as errors

Reply:
Find with me.

There is however a `(profile release)` stanza in the `dune-workspace` file which should do the same.